hack/ldapsearch/README.md

```diff
@@ -0,0 +1,19 @@
+# ldapsearch.sh
+
+Translate your `LDAPIdentityProvider` into `ldapsearch` queries for debugging purposes.
+
+Usage
+
+```shell
+kubectl get ldapidentityprovider <name> \
+  --namespace=<namespace> \
+  --output=yaml | ./hack/ldapsearch/ldapsearch.sh
+```
```

Comment:
Lets make the script name more clear.  `./hack/ldapsearch/ldapsearch.sh` ->. `./hack/ldapsearch/identity-provider-to-search-query.sh` or something.  I don't know that terse is as important as clarity given the use case.

Reply:
I changed the usage. Also removed the readme. Didn't feel like it deserved its own readme. Instead, I put comments at the top of the file like we have for other hack scripts.